lib/main.php

```diff
@@ -119,10 +119,10 @@ function convert ($body, $query_string, $set_properties, $group_id = null) {
       }
 
       // text description
-      if (isset($body['body_text'])) {
-        $entry['description'] = $body['body_text'];
-      } else if (isset($body['body_html'])) {
+      if (isset($body['body_html'])) {
         $entry['description'] = strip_tags($body['body_html']);
+      } else if (isset($body['body_text'])) {
```

Comment:
@matheusgnreis  eu acho q verificar se o `body_text` não está `empty()` aqui vai evitar isto aqui com os produtos existentes e sincronizados via bling.
https://community.e-com.plus/t/produtos-rejeitados-por-falta-de-descricao-no-catalogo-do-facebook/718/3
```suggestion
      if (isset($body['body_html'])) {
        $entry['description'] = strip_tags($body['body_html']);
      } else if (isset($body['body_text']) && !empty($body['body_text'])) {
```